usl_pipeline/study_area_uploader/study_area_uploader/transformers/soil_classes_transformers.py

```diff
@@ -50,6 +57,9 @@ def transform_soil_classes_as_green_areas(
         elevation_header, green_areas_polygons
     )
     logging.info("Soil classes raster is getting created...")
+    # Raster for soil classes is made based on soil_classes_polygons that might include
+    # non-green area polygons. This is done specifically so that the cells of those
+    # non-green classes wouldn't be gap-filled.
```

Comment:
what do you mean by 'gap-filled' here?

Reply:
I mean filled-in in place of absent data (for raster cells covered by non-green soil classes we should NOT do nearest-neighbor filling-in). Do you have any suggestions for how to formulate it better?

Reply:
Maybe, 
"This is to avoid creating cells inside the green spaces with missing soil values, which would be interpolated, when we need to leave them as their non-green-area soil type."
?

Reply:
SG, made a correction.